File: java/logica_y_validaciones/validar_campos_obligatorios.java
package logica_y_validaciones;
public class validar_campos_obligatorios {
    public static void main(String[] args) {
        String nombre = "Ana Gómez";
        String telefono = "3001234567";
        String correo = "ana@gmail.com";
        if (!nombre.isEmpty() && !telefono.isEmpty() && !correo.isEmpty()) {
            System.out.println("Todos los campos son obligatorios y están completos.");
        } else {
            System.out.println("Error: hay campos obligatorios vacíos.");
        }
    }
}

File: visual_basic/condicionales/validar_campos_obligatorios/validar_campos_obligatorios.vb
Imports System
Module validar_campos_obligatorios
    Sub Main(args As String())
        Dim nombre As String
        Dim telefono As String
        Console.Write("Ingrese el nombre: ")
        nombre = Console.ReadLine()
        Console.Write("Ingrese el teléfono: ")
        telefono = Console.ReadLine()
        If nombre = "" Or telefono = "" Then
            Console.WriteLine("Error: Todos los campos son obligatorios ⚠️")
        Else
            Console.WriteLine("Datos guardados correctamente ✅")
            Console.WriteLine("Nombre: " & nombre)
            Console.WriteLine("Teléfono: " & telefono)
        End If
    End Sub
End Module